damage_calculator.py

```diff
@@ -251,11 +251,11 @@ def calc_auto_dmg(god, power):
 
     return attDamage
 
-
-def calc_dps(client, god, build, enemy, enemy_build, enemy_level, level=20):
+def calc_dps(client, god, build, enemy, enemy_build, enemy_level, number_of_autos, level=20):
```

Comment:
can you set the default number_of_autos to 10

number_of_autos=10 in this line

Reply:
donee